src/content/api/logging.md

```diff
@@ -21,19 +22,19 @@ Benefits of custom logging API in webpack:
 - CLI, UI tools for webpack may choose different ways to display logging
 - webpack core can emit logging output, e.g. timing data
 
-By introducing webpack logging API we hope to unify the way webpack plugins and loaders emit logging messages and allow better ways to inspect build problems. Integrated logging solution supports plugins and loaders developers by improving their development experience. Paves the way for non-CLI webpack solutions like dashboards or other UIs.
+By introducing webpack logging API we hope to unify the way webpack plugins and loaders emit logs and allow better ways to inspect build problems. Integrated logging solution supports plugins and loaders developers by improving their development experience. Paves the way for non-CLI webpack solutions like dashboards or other UIs.
```

Comment:
i dont think this is an improvement, plugins and loaders emit messages and webpack logs them into terminal via logger right?

Reply:
According to lines above it.
The plugins and loaders emit logs they want and webpack emit these based on usage of api, it may be emitted to a terminal (for CLIs) or to a UI.

Reply:
Right, and by logs you mean what exactly? Messages?

Reply:
Yes logged messages only

Reply:
Eugene may be, here is what I meant.
By logs I mean any statistical or build related information that plugins and loaders want to emit.
Rest you may see what fits here.